### PR TITLE
fix: remove initial range queryParam for Explore page

### LINKED ISSUE #2568: Bug: Can't navigate back in the browser from the Explore page
Currently a querystring parameter gets added to https://beta.app.opensauced.pizza/javascript/dashboard/filter/recent in a `useEffect`, i.e. `?range=30`. When you navigate back in the browser, it returns to https://beta.app.opensauced.pizza/javascript/dashboard/filter/recent, but the `useEffect` kicks in again and brings you back to https://beta.app.opensauced.pizza/javascript/dashboard/filter/recent?range=30 .

The default range is 30 days, so there's no need to add it on with a `useEffect`.

### PR DESCRIPTION
**Pull request by zeucapua**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [X] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2568 

## Steps to QA

1. Go to home page
2. Click the `Explore` button in navigation bar
3. Check that there is no `?range=30` at the end of URL when loading page
4. Navigate back from browser

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Are you ready

### CODE REVIEW

File: components/organisms/ToolList/nav.tsx [hunk 1]
```diff
@@ -53,16 +49,13 @@ const Nav: React.FC<NavProps> = ({ toolList, selectedTool = "dashboard", selecte
           />
         </div>
       ))}
-      {range ? (
-        <div className="ml-auto hidden md:block">
-          <ComponentDateFilter
-            setRangeFilter={(selectedRange) => {
-              setQueryParams({ range: `${selectedRange}` });
-            }}
-            defaultRange={Number(range)}
```
Comment (nickytonline) [inline components/organisms/ToolList/nav.tsx:62]: Refactor (blocking): We need to keep the `defaultRange` because if you navigate to e.g. https://deploy-preview-2573--oss-insights.netlify.app/javascript/dashboard/filter/recent?range=90, it will load the default of 30 days.

TLDR; Keep this file as it was, but add a default value if none is in the querystring, i.e.

```typescript
const { range = 30 } = router.query;
```

**Review by nickytonline — CHANGES_REQUESTED** [review]
This is working great @zeucapua. One small change request.

**Review by nickytonline — APPROVED** [review]
Great stuff @zeucapua!

:shipit:

**Review by jpmcb — APPROVED** [review]
👏🏼
